prow/cmd/deck/static/prow/prow.ts

```diff
@@ -7,6 +7,7 @@ import {JobHistogram, JobSample} from './histogram';
 declare const allBuilds: Job[];
 declare const spyglass: boolean;
 declare const rerunCreatesJob: boolean;
+declare const csrfToken: string;
```

Comment:
Instead of including this in every template, I'd rather see it in the base template. Then it could be handled largely automatically between `prepareBaseTemplate` and `handleSimpleTemplate`.

Reply:
I'm not sure how to do this. Would this mean passing the token into the base template and still including it as a constant in the ts files?